### PR TITLE
Update description in lxc file mount

### PR DESCRIPTION
**Pull request by tenforward**
Signed-off-by: KATOH Yasufumi <karma@jazz.email.ne.jp>

### CODE REVIEW

File: lxc/file.go [hunk 1]
```diff
@@ -912,7 +912,7 @@ func (c *cmdFileMount) Command() *cobra.Command {
 		`Mount files from instances`))
 	cmd.Example = cli.FormatSection("", i18n.G(
 		`lxc file mount foo/root fooroot
-   To mount /root from the instance and onto the local fooroot directory.`))
+   To mount /root from the instance foo and onto the local fooroot directory.`))
```
Comment (tomponline) [inline lxc/file.go:915]: Please can you remove the "and" after "foo"? Thanks

Comment (ykato-idcf) [reply lxc/file.go:915]: OK!!

Comment (tenforward) [reply lxc/file.go]: Oops, I forgot make i18n ... wait a moment.

**Comment by tomponline** [comment]
Thanks!

**Pull request merged**